app/src/test/java/io/apicurio/registry/auth/AuthTestAuthenticatedReadAccess.java

```diff
@@ -56,21 +58,24 @@
 
     final String groupId = getClass().getSimpleName() + "Group";
 
+    ApicurioHttpClient httpClient;
```

Comment:
How is this closed at the end of the test?

Reply:
It's not. Actually, I'm releasing (but failing) a new version of the http client library where the implementation of the close method is empty since I'm making the jdk-based client use its default executor and everything is working as expected (I have verified it locally). Some of the code on this PR is just sanity checks in the case in the future the implementation is actually doing something. I think it's fine to leave tests as they are.